ansible/roles/test/vars/testcases.yml

```diff
@@ -58,6 +58,11 @@ testcases:
       topologies: [t0, t0-64, t0-64-32, t0-116]
       required_vars:
           ptf_host:
+    dtel:
+      filename: dtel.yml
+      topologies: [ptf32]
```

Comment:
How about ptf64 topology?

Reply:
Dataplane telemetry tests require only 3 ports for now. So, we use only ptf32 topology